File: Python/Cubiculo.py
class Cubiculo:
    _id_contador = 0
    
    def __init__(self, cantidad, ancho, largo, estado, tipo):
        Cubiculo._id_contador+=1
        self.id = Cubiculo._id_contador
        self.cantidad = cantidad
        self.ancho = ancho
        self.largo = largo
        self.estado = estado
        self.tipo = tipo

    # Metodos Getters
    def get_id(self):
        return self.id
    
    def get_cantidad(self):
        return self.cantidad

    def  get_ancho(self):
        return self.ancho

    def get_largo(self):
        return self.largo
 
    def get_estado(self):
        return self.estado
    
    # Metodos Setters
    def set_cantidad(self, cantidad):
        self.cantidad = cantidad
    
    def set_ancho(self, ancho):
        self.ancho = ancho

    def set_largo(self, largo):
        self.largo = largo
    
    def set_estado(self, estado):
        self.estado = estado
    
    def __str__(self):
        return (f"Parqueadero(id={self.id}, "
                f"nombre='{self.nombre}', area='{self.area}', "
                f"cubiculo='{self.cubiculo}', direccion='{self.direccion}', "
                f"estado='{self.estado}')")
        
    
    
    

File: Java/Cubiculo.java
public class Cubiculo {
    private static int contadorId;
    private int id;
    private int cantidad;
    private double ancho;
    private double largo;
    private String estado;
    private String tipo;

    public Cubiculo(int cantidad, double ancho, double largo, String estado, String tipo) {
        this.id = ++Cubiculo.contadorId;
        this.cantidad = cantidad;
        this.ancho = ancho;
        this.largo= largo;
        this.estado = estado;
        this.tipo = tipo;

    }


    public int getId(){
        return this.id;
    }
    public int getCantidad() {
        return cantidad;
    }

    public void setCantidad(int cantidad) {
        this.cantidad = cantidad;
    }

    public double getAncho() {
        return ancho;
    }

    public void setAncho(double ancho) {
        this.ancho = ancho;
    }

    public double getLargo() {
        return largo;
    }

    public void setLargo(double largo) {
        this.largo = largo;
    }

    public String getEstado() {
        return estado;
    }

    public void setEstado(String estado) {
        this.estado = estado;
    }

    public String getTipo() {
        return tipo;
    }

    public void setTipo(String tipo) {
        this.tipo = tipo;
    }

    



    
    

}
